### ISSUE TITLE
Problems with the new Yutube interface in Edge

### ISSUE THREAD

**Issue by gdata1** [nvaccess/nvda#8098]
Hello guys. I am observing the following problems with Yutube in edge.
1. With the new interface Yutube NVDA works slower than with the old version.
2. NVDA does not read the name video if you go through the headers using the H key.
This happens in edge.windows 10 1803 17123

NVDA next-14952,dd1cbe50

**Comment by leonardder** [comment]
Thanks for your report. Are you able to reproduce this issue only with nVDA, or also with Windows Narrator on this version of Windows/Edge?

**Comment by gdata1** [comment]
These problems are only used when using NVDA.
You can also play them on Windows 10 1709.

**Comment by Adriani90** [comment]
This works as expected in Edge Chromium 81 with NVDA Version: alpha-20026,16ec967e. Closing as works for me.
